src/styles/layout/index.md.njk

```diff
@@ -13,7 +13,7 @@ show_page_nav: true
 
 Design for small screens first, starting with a single-column layout.
 
-For most types of page, we recommend using either a 'two-thirds' or a 'two-thirds and one third layout'. That stops lines of text getting so long that the page becomes difficult to read on desktop devices. This would usually mean no more than 75 characters per line.
+For most types of page, we recommend using either a 'two-thirds' or a 'two-thirds and one-third layout'. That stops lines of text getting so long that the page becomes difficult to read on desktop devices. This would usually mean no more than 75 characters per line.
```

Comment:
```suggestion
For most types of page, we recommend using either a 'two-thirds' or a 'two-thirds and one-third' layout. That stops lines of text getting so long that the page becomes difficult to read on desktop devices. This would usually mean no more than 75 characters per line.
```